### PR TITLE
Airflow.tree schedule_interval offset fix

### PR DESCRIPTION
**Pull request by microblag**
PR for Issue #225

### CODE REVIEW

File: airflow/www/app.py [hunk 1]
```diff
@@ -993,13 +993,19 @@ def tree(self):
         else:
             base_date = dateutil.parser.parse(base_date)
 
+         start_date = dag.start_date
```
Comment (mistercrunch) [inline airflow/www/app.py]: indent looks off

**Comment by mistercrunch** [comment]
Thanks for the PR, sorry about the delay, reviewing today

**Comment by mistercrunch** [comment]
This looks perfect, it fixed issues someone had internally with a weekly DAG. 
img

Thank you for your contribution!

**Comment by microblag** [comment]
Glad to help!

**Pull request merged**